### ISSUE TITLE
OcSearchBar has no border anymore

### ISSUE THREAD

**Issue by kulmann** [owncloud/owncloud-design-system#1870]
The OcSearchBar has no border anymore, hence is not accessible anymore (color contrast ratio of search bar on light backgrounds not sufficient)

**Comment by pascalwengerter** [comment]
Arguably was broken before already but sure let's collect a lot of small glitches here and fix them soon! Icons on https://owncloud.design/#/oC%20Components/OcResource (examples) seem broken

**Comment by kulmann** [comment]
Let me reword the issue then, sorry. Didn't want to play blame games 😉
